### ISSUE TITLE
Disease clouds - buff?

### ISSUE THREAD

**Issue by ByBurton** [Sh0wdown/UnofficialCrusaderPatch#187]
So I had been thinking about your next incoming change, the healers.
They have been buffed, a lot. How about increasing the time a disease cloud from throwing cows stays?

**Comment by Wolfram92** [comment]
Disease clouds in Skirmish (by throwing cows) are kinda weak in general. They just stay for a short amount of time, have a fairly small radius and it's quite easy to move your troops around.

Also, throwing cows on towers and walls doesn't work at all, which makes many attacks of an opponent like the Wolf useless. When I play against him, he always tries to throw Cows on the towers and walls, instead at the people behind it.

**Comment by ByBurton** [comment]
I doubt that it will be easy to make disease clouds reach on to walls, or if that would actually be a desirable thing.
But we can agree that they need to be buffed. 
Factors that can be buffed:
-Time the cloud poison stays in effect
-Radius / Range of the cloud disease effect
-Damage tick (this one is fine the way it is now)

**Comment by Panbutt** [comment]
Yeah, I'm in agreement that disease clouds need to linger much much longer. Maybe even grow stronger the longer they stay before eventually dying out, to at least vaguely simulate a disease outbreak. It'd make apothecaries much more useful to combat it.

Right now cow corpses just elicit a 'welp better move my swordsmen five steps to the right' sort of response. I think there needs to be a real sense of terror when the enemy starts flinging corpses at you. An actual risk of some sort of lingering outbreak that could potentially kill everyone in your castle if you're not prepared.

**Comment by Monsterfisch** [comment]
Why not have the people that walk through it get a poison effect that lasts for a certain time unless the person is healed damaging them over time? that way it would be a more consistent damage output and be a lot more effective when throwing the cows at crowded places. one could argue that the poison would have a chance to spread to nearby peasants and units as well 5% chance or something like that simulating an outbreak. (would have to be tested heavily through)

**Comment by Monsterfisch** [comment]
mhm sad times 
I could already imagine a peasant/unit walk around with a small poison cloud following him around until dead/times up/healed with the cloud infecting other peasants (with a low chance) when crossing that units/peasants path

**Comment by Heroesflorian** [comment]
> mhm sad times

Indeed! 

There are no other real DoT sources ingame yet, right? Except fire of course, which does DoT while standing on it... hmm... 

> I could already imagine a peasant/unit walk around with a small poison cloud following him around until dead/times up/healed with the cloud infecting other peasants (with a low chance) when crossing that units/peasants path

Considering fire creates sparks that might "infect" (ignite) other nearby tiles (and buildings on those tiles)... I think "poison sparks" would probably be less complex to implement than moving DoT effects per unit.

**Comment by Monsterfisch** [comment]
DoT sources are as you said disease and fire now to realize my idea "all that needs to be done" would be that as soon as a person enters a cloud  one of the 4 (maybe more in the future) clouds would start following him around damaging him over time and if the person gets close to other people have a chance to give them a copy of the cloud that is following him around (and so on), until he is dead the timer on the cloud ran out or he is healed.

**Comment by Wolfram92** [comment]
> Right now cow corpses just elicit a 'welp better move my swordsmen five steps to the right' sort of response. I think there needs to be a real sense of terror when the enemy starts flinging corpses at you. An actual risk of some sort of lingering outbreak that could potentially kill everyone in your castle if you're not prepared.

This! Maybe it would be possible to implement the pestilence mechanic from the campaigns in Skirmish. The opponent throws a cow behind the walls and disease will start to spread. During the event in campaign mode, clouds start popping up everywhere in the castle. Perhaps it's possible to use the mechanic from that event in Skirmish, but without the drop in popularity,

**Comment by Heroesflorian** [comment]
Or... (just thinking out loud here, might not be practicable or hard to balance) ...skip the disease spreading if too complex to implement, but then DO include a popularity drop instead. Cannot kill the enemy with diseased cows? Then just annoy the hell outta him and cripple his economy with unhappy peasants demanding for increase food rations or lower taxes! 
**\o/**

**Comment by Monsterfisch** [comment]
Could give fresh peasants a 1 min immunity/damage reduction by default to prevent the disease to completely flatten the economy from one cow.
Since the % of the illness spreading would be rather low you would, have to bombard the castle with cows to have it actually go really out of control. (needs to be tested ofc to actually know it would work or not regardless so I imagine it's not gonna happen any time soon if at all)

**Comment by PitchNeeded** [comment]
The proposed change sounds more realistic, but it would make it easier for a human lord against the AI. To prevent this, I would like it to be made (if possible) that the AI will actively try and move their units away from a disease cloud. For example, if I am attacking an Arabian AI with horse archers below their keep, if I throw cows below the keep and a gas cloud appears, the horse archers will try and move away.
